src/i18n/strings.ftl

```diff
@@ -751,6 +751,8 @@ Navigates-to-AI-camera = Navigates to AI camera
 Navigates-to-bulk-importer = Navigates to bulk importer
 Navigates-to-camera = Navigates to camera
 Navigates-to-explore = Navigates to explore
+Navigates-to-log-in-screen = Navigates to log in screen
```

Comment:
It is always good to provide context to strings in this file, especially for strings that are not directly user-facing, so everything a11y related. E.g.:
`# Accessibility hint for ...` one line above.

These comments appear in Crowdin were we crowdsource our translations. So, translators will see this context as well.